Refactor to pyansys standards (#140)
pyansys/pydpf-post: README.md

etc.). Various operators are available allowing the manipulation and
the transformation of this data.

> The Python `ansys-dpf-post` package provides an simplified Python

Reviewer: ```suggestion
The Python `ansys-dpf-post` package provides a simplified Python
```

> interface to DPF, thus enabling rapid post-processing, without ever

Reviewer: ```suggestion
interface to DPF, thus enabling rapid postprocessing without ever
```

leaving a Python environment. 

This module leverages the DPF-Core project's ``ansys-dpf-core`` package and can
be found by visiting [PyDPF-Core
GitHub](https://github.com/pyansys/pydpf-core).  Use ``ansys-dpf-core`` for

> building more advanced and customized workflows using Ansys's DPF.

Reviewer: @akaszynski Grammatically, Ansys's DPF is correct. However, all corporate content these days is using Ansys' as the possessive form. (This is wrong because you naturally pronounce as "Ansys-es".) Should we leave as is or reflect what is put out by our corporate communications team?

> building more advanced and customized workflows using Ansys's DPF.

Author: > Jenna told me that it had to be updated in TFS and pushed to the repo?

I think we're moving all dev for DPF into GitHub (@PProfizi, please verify).

Leave as-is and then commit to a follow-up PR.

multi-gigabyte models in a short script.  DPF-Post also details the
usage of the operators used when computing the results so you can also
build your own custom, low level scripts using the
[DPF-Core](https://github.com/pyansys/pydpf-core) module.

### License

> ``PyDPF-Post`` is licensed under the MIT license.  Please see the
> [LICENSE](https://github.com/pyansys/dpf-post/raw/master/LICENSE) for more
> details.

Reviewer: ```suggestion
``PyDPF-Post`` is licensed under the MIT license.  For more information, see the
[LICENSE](https://github.com/pyansys/dpf-post/raw/master/LICENSE).
```
